mercury_app/utils.py

```diff
@@ -514,10 +514,23 @@ def get_json_donut(percentage, name, id_int):
 
 def get_summary_types_handed(order_ids):
     total_mercha = Merchandise.objects.filter(order_id__in=order_ids).values(
-        'name').annotate(name_count=Count('name')).order_by('name')
-    handed_mercha = Merchandise.objects.filter(
-        order_id__in=order_ids, delivered=True).values(
-        'name').annotate(name_count=Count('name')).order_by('name')
+             'name').annotate(name_count=Sum('quantity')).order_by('name')
+    types = Merchandise.objects.filter(order_id__in=order_ids).values_list('name', flat=True).distinct()
+    types_id = []
+    for ty_pe in types:
+        types_id.append(Merchandise.objects.filter(order_id__in=order_ids, name=ty_pe))
+    ids_mercha = Merchandise.objects.filter(order_id__in=order_ids)
+    type_transaction_count = []
+
+    for ty_pe in types_id:
+        total_for_type = 0
+        for item in ty_pe:
+            total_for_type += Transaction.objects.filter(merchandise=item, operation_type='HA').count()
```

Comment:
Maybe you can use some more Pythonic way to sum these, like:
`total_for_type = sum(Transaction.objects.filter(merchandise=item, operation_type='HA').count() for item in ty_pe)`
i saw you made this kind of sums many times, i hope it helps